src/main/java/com/powsybl/openloadflow/ac/IncrementalTransformerVoltageControlOuterLoop.java

```diff
@@ -33,8 +36,10 @@ public class IncrementalTransformerVoltageControlOuterLoop extends AbstractTrans
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IncrementalTransformerVoltageControlOuterLoop.class);
 
-    private static final int MAX_TAP_SHIFT = 3;
+    public static final int DEFAULT_MAX_TAP_SHIFT = 3;
```

Comment:
Maybe we could increase to 5?

Reply:
Without any tests, maybe we should leave it like this...